compiler/noirc_frontend/src/hir/def_collector/dc_crate.rs

```diff
@@ -358,6 +373,47 @@ impl DefCollector {
     }
 }
 
+fn inject_prelude(
+    crate_id: CrateId,
+    context: &mut Context,
```

Comment:
```suggestion
    context: &Context,
```

Do we use this mutability?